### DISCUSSION TITLE
Set Size of Video Player without taking both sides

### DISCUSSION THREAD

**Discussion by chuntimchan** [PaulleDemon/tkVideoPlayer#40] [Q&A] [answered]
Hey,

I'm trying to create a simple viewing app in python as my first project.
I can't seem to figure out how to resize the video player though. I've tried set_size but that only changes the video size and not the window they video is playing on.

It looks something like this so I can't just use Pack and have it fill in both sides and have buttons on the bottom.
![image](https://github.com/PaulleDemon/tkVideoPlayer/assets/79533731/0488f75f-8d46-4a83-a7ff-d761c89aed27)

I even tried to make a new frame and put the video in it to no avail.
![image](https://github.com/PaulleDemon/tkVideoPlayer/assets/79533731/93fefced-5467-4991-af1a-411c94780625)

**Comment by PaulleDemon** [comment]
@chuntimchan does https://github.com/PaulleDemon/tkVideoPlayer/discussions/19 answer your question?

**Reply by chuntimchan** [reply]
Hi @PaulleDemon unfortunately not, I've already read that and I couldn't solve the issue

**Reply by PaulleDemon** [reply, marked as answer]
Specify the height and width in `.place(x=300, y=100, width=1000, height=1000)`
```py
import tkinter as tk
from tkVideoPlayer import TkinterVideo
root = tk.Tk()


frame = tk.Frame(root, width=1000, height=100)
frame.place(x=300, y=100, width=1000, height=1000) # <-- notice here

player = TkinterVideo(frame, scaled=True)
player.load("test2.mp4")
player.pack(expand=True, fill="both")
player.play()
root.mainloop()
```

**Reply by chuntimchan** [reply]
Thank you!
